scipy/interpolate/_cubic.py

```diff
@@ -409,14 +393,6 @@ class Akima1DInterpolator(CubicHermiteSpline):
         based on first and last intervals, or to return NaNs. If None,
         ``extrapolate`` is set to False.
 
-    Methods
-    -------
-    __call__
-    derivative
-    antiderivative
-    integrate
-    roots
```

Comment:
This one was deliberate: `PPoly.extend` is inherited by does not construct an Akima spline :-(. Ditto for `from_spline`: it is inherited to create a PPoly instance which may or may not be monotone. 

Ditto for PCHIP.

Reply:
OK, but what about the `solve` method? Those should work, right?